contracts/ERC20StakingRewardsDistribution.sol

```diff
@@ -158,25 +170,55 @@ contract ERC20StakingRewardsDistribution {
         );
     }
 
-    function cancel() external onlyOwner {
+    function cancel() external override onlyOwner {
         require(initialized && !canceled, "SRD19");
         require(block.timestamp < startingTimestamp, "SRD08");
+        canceled = true;
+        emit Canceled();
+    }
+
+    function recoverRewardsAfterCancel(address _token)
```

Comment:
Shouldn't have a `_token` here, it's unused

Reply:
Solved in 988c11d7179a8d3fef4b6f5421e6945236c5e5cc